api/src/main/java/de/valtech/aecu/api/groovy/console/bindings/filters/FilterBy.java

```diff
@@ -35,6 +35,6 @@
      * @param resource resource
      * @return matches
      */
-    boolean filter(@Nonnull Resource resource);
+    boolean filter(@Nonnull Resource resource, @Nonnull StringBuffer stringBuffer);
```

Comment:
Breaking change, requires to change version to 2.0 in package-info.java.

Reply:
done